docs/components/checkboxes.md

```diff
@@ -0,0 +1,225 @@
+<!--docs:
+title: "Material selection controls: CheckBoxes"
+layout: detail
+section: components
+excerpt: "Selection controls allow the user to select options."
+iconId:
+path: /catalog/SelectionControlsCheckBoxes/
+-->
+
+# Selection controls: checkboxes
+
+[Selection controls](https://material.io/components/selection-controls#usage) allow the user to select options.
+
+Use checkboxes to:
+
+- Select one or more options from a list
+- Present a list containing sub-selections
+- Turn an item on or off in a desktop environment
+
+![Checkbox hero example for menu options](assets/checkboxes/checkbox-hero.png)
+
+## Contents
+
+- [Using checkboxes](#using-checkboxes)
```

Comment:
Add a contents link and one for all 2-level headers

Reply:
Done!